react/components/Table/SimpleTable.js

```diff
@@ -215,16 +260,23 @@ class SimpleTable extends Component {
                               fullWidthColWidth
                             ),
                           }}
-                          className={`flex items-center w-100 h-100 truncate ph4 ${
-                            disableHeader && rowIndex === 0 ? 'bt' : ''
-                          } bb b--muted-4 ${
+                          className={`flex items-center w-100 h-100 ph4 bb 
+                            b--muted-4 truncate ${
+                              disableHeader && rowIndex === 0 ? 'bt' : ''
+                            } ${
                             onRowClick && rowIndex === hoverRowIndex
                               ? 'pointer bg-near-white c-link'
                               : ''
                           } ${columnIndex === 0 && fixFirstColumn ? 'br' : ''}`}
                           onClick={
-                            onRowClick
-                              ? e => onRowClick({ e, index: rowIndex, rowData })
+                            onRowClick &&
+                            property !== '_VTEX_Table_Internal_lineActions'
+                              ? e =>
```

Comment:
⛏ `event` would be nicer, but its your call

Reply:
@lbebber I'm not sure I understand what you are suggesting 🤔

Reply:
renaming from `e` to `event`, to make it clearer. just a suggestion though

Reply:
@ericreis swap `e` for `event` for easier understanding in the callback object param.